tuneit/tools/base.py

```diff
@@ -87,7 +87,7 @@ def record(self, value):
             self._trials = None
         elif value is True:
             data_keys = [
-                key[: key.rfind("-")] + key[key.rfind("__") :]
+                key[: key.rfind("-")] + key[key.rfind("_") :]
```

Comment:
I saw this now.. I don't think is the best way to remove the hash of the key.
I would have instead an option inside get_info that removes the hash directly, e.g.
```python
    def get_info(self, short=False):
        all_info = {}
￼        for data in self.datas:
￼            info = self[data].get_info()
￼            for key in list(info):
￼                name = data[: data.rfind("-")] if short else data
￼                all_info[f"{name}_{key}"] = info[key]
￼        return all_info
```